backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -33,13 +33,22 @@
 import com.datastax.driver.core.Session;
 
 public final class CassandraCluster implements AutoCloseable {
```

Comment:
Add javadoc expplaining that you rely on a super user for init, and provide a specific user with restrictive write for testing please.